src/wdio/services/TerraCommands/hide-input-caret.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
+  if (!global.browser.isExisting(selector)) {
+    throw new Error(`No element could be found with the selector '${selector}'.`);
+  }
+  global.browser.execute(`document.querySelector("${selector}").style.caretColor = "transparent";`);
```

Comment:
Is it possible to wrap this in a try-catch so this browser check isn't always run for valid cases? something like:

```
try {
  global.browser.execute(`document.querySelector("${selector}").style.caretColor = "transparent";`);
} catch (error) {
 if (!global.browser.isExisting(selector)) {
    throw new Error(`No element could be found with the selector '${selector}'.`);
  } else { 
    throw new Error(error)
  }
}
```

Reply:
Changed in cce5091cbdebce172e5f5c54d0950f86134a8b33